### PR TITLE
IoT page

### PR DESCRIPTION
**Pull request by Lukewh**
# Done
Fixes https://github.com/canonicalltd/snapcraft-design/issues/502
Added IoT page

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/iot
- Compare to the [copy doc](https://docs.google.com/document/d/1yfnQ5pXu5tDTG3M7Yv_gpjd894NUzSBKcvhB_pZqrMo/edit#)

### CODE REVIEW

**Comment by therealjuan** [comment]
Would it be possible to include more padding on the cards?

image

Can we include IoT as part of the navigation? Later on we will expand when we have more industries but this content is lonely now...

**Comment by Greggless** [comment]
We'll need to get rid of 'libraries' from the 'IoT libraries tools, and servers' section.

**Comment by lyubomir-popov** [comment]
@Lukewh Taking out the hrule under the card results in:
• lack of clarity about which section the testimonial is for 
• different space above networking

I would rather stick to one style of p-card - and in this case I would prefer the highlighted one, because it sits well between the hrules above underneath:

screen shot 2018-08-08 at 11 09 00

**Comment by bartaz** [comment]
Isn't it officially  'Internet of Things' rather then 'Internet of things'?

**Comment by bartaz** [comment]
There is a strange additional dotted border on top of the cards in the bottom:

screen shot 2018-08-08 at 12 35 49

**Comment by therealjuan** [comment]
> Isn't it officially 'Internet of Things' rather then 'Internet of things'?

That's how normally the term is referred in media and that is why the acronym is IoT. See examples [#1](https://www.wired.co.uk/article/internet-of-things-what-is-explained-iot), [#2](https://www.forbes.com/sites/jacobmorgan/2014/05/13/simple-explanation-internet-things-that-anyone-can-understand/#317e58421d09), [#3](https://channel9.msdn.com/Shows/Internet-of-Things-Show/IoT-In-Action-The-Next-Agricultural-Revolution), [#4](https://www.techopedia.com/definition/28247/internet-of-things-iot)

**More on IoT & Snapcraft** can do with some spacing from the HR line

**Comment by Lukewh** [comment]
@therealjuan @bartaz should I change it? If so we should change it in the copy doc too... That's what I'm using as the source of truth

File: templates/store/categories/iot.html [hunk 1]
```diff
@@ -0,0 +1,456 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}{% endblock %}
```
Comment (bartaz) [inline templates/store/categories/iot.html]: Please link the copy doc here: https://docs.google.com/document/d/1yfnQ5pXu5tDTG3M7Yv_gpjd894NUzSBKcvhB_pZqrMo/edit

File: templates/store/categories/iot.html [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+      </div>
+    </div>
+
+    <div class="row is-bordered">
+      <h3 class="p-heading--four">More on IoT & Snapcraft</h3>
+    </div>
+    <div class="row">
+      <div class="u-equal-height">
+        <div class="col-4 p-card--post">
+          <div class="p-card__content">
```
Comment (bartaz) [inline templates/store/categories/iot.html]: This `.p-card__content` seems to be having unnecessary top dotted border.

screen shot 2018-08-09 at 12 49 46

Possibly it's a vanilla issue? 
screen shot 2018-08-09 at 12 50 38

Comment (bartaz) [reply templates/store/categories/iot.html]: I just noticed that we have the same issue on `/blog`:

screen shot 2018-08-09 at 12 56 29

Comment (bartaz) [reply templates/store/categories/iot.html]: Reported separately: https://github.com/canonical-websites/snapcraft.io/issues/1004

**Comment by bartaz** [comment]
Should not be merged before `IoT` is added to nav in build and docs.

File: templates/store/categories/iot.html [hunk 3]
```diff
@@ -0,0 +1,456 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1yfnQ5pXu5tDTG3M7Yv_gpjd894NUzSBKcvhB_pZqrMo/edit{% endblock %}
+
+{% block meta_title %}
+  Snapcraft - Fast and secure publishing for Internet of Things
```
Comment (caldav) [inline templates/store/categories/iot.html]: Suggestion: "Publish, install and update IoT apps - Snapcraft"

File: templates/store/categories/iot.html [hunk 4]
```diff
@@ -0,0 +1,456 @@
...
+
+{% block meta_description %}
+  Transactional updates, airtight security, and compatibility across architectures and OSes make snaps a perfect fit for IoT.
+{% endblock %}
+
+{% block content %}
+  <section id="main-content" class="p-strip--image snapcraft-banner-background">
+    <div class="row">
+      <div class="col-7">
+        <h2>Internet of Things</h2>
```
Comment (caldav) [inline templates/store/categories/iot.html]: This page needs a h1 tag, using `class="p-heading--two"` will allow preserving the current styling.

File: templates/store/categories/iot.html [hunk 5]
```diff
@@ -0,0 +1,456 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1yfnQ5pXu5tDTG3M7Yv_gpjd894NUzSBKcvhB_pZqrMo/edit{% endblock %}
+
+{% block meta_title %}
+  Snapcraft - Fast and secure publishing for Internet of Things
+{% endblock %}
+
+{% block meta_description %}
+  Transactional updates, airtight security, and compatibility across architectures and OSes make snaps a perfect fit for IoT.
```
[the remark this answers is not in the document]
Comment (Lukewh) [reply templates/store/categories/iot.html]: Done, thanks

File: templates/store/categories/iot.html (hunk 3 above)
Comment (Lukewh) [reply templates/store/categories/iot.html]: Done, thanks

File: templates/store/categories/iot.html (hunk 4 above)
Comment (Lukewh) [reply templates/store/categories/iot.html]: Done, thanks

**Comment by Lukewh** [comment]
Merging as the page needs to be there before other sites get deployed!

**Pull request merged**